modules/theme/src/themes/wso2is/apps/developer-portal.overrides

```diff
@@ -1887,3 +1887,16 @@ TODO : Move values to separate variables and cleanup styles.
     color: @primaryColor !important;
     text-transform: capitalize !important;
 }
+
+/*-------------------------------
+         Sign-in methods
+     Add Authenticator Modal
+--------------------------------*/
+
+.add-authenticator-modal {
+    .authenticator-container {
+        .resource-grid-wrapper .ui.cards.resource-grid .ui.card.info-card {
+            width: 290px;
```

Comment:
Could you please check whether setting fixed widths can casue distorted UIs in medium size screens as well?

Reply:
Recently, width of the resource cards are increased from 290px to 330px. But this new size is not suitable for resource cards within wizards. Therefore, I revert the width to 290px only for the resource cards within the wizards. Since I don't introduce any new things here, it should not break medium sizes as well.